action.yml

```diff
@@ -14,6 +14,11 @@ inputs:
   yarn-version:
     description: 'Version Spec of the yarn version to use.  Examples: 1.6.x, 10.15.1, >=10.15.0'
     default: ''
+  registry-url:
+    description: 'Optional registry to set up for auth. Will set the registry in a project level .npmrc and .yarnrc file, and set up auth to read in from env.NODE_AUTH_TOKEN'
```

Comment:
this mentions that it writes a `.yarnrc` file, but as far as I can tell there is no code to do that

Reply:
I copied this from the original implementation, it doesn't appear to actually do anything with yarnrc so I just removed that